mlir/include/Quantum/Transforms/Passes.td

```diff
@@ -117,6 +117,20 @@ def MergeRotationsPass : Pass<"merge-rotations"> {
     let options = QuantumCircuitTransformationPass.options;
 }
 
+def PropagateSimpleStatesTesterPass : Pass<"propagate-simple-states-tester"> {
+    let summary = "Propagate simple states through their corresponding gates down the circuit. The simple states are the six Pauli basis states.";
+
+    let constructor = "catalyst::createPropagateSimpleStatesTesterPass()";
+    let options = QuantumCircuitTransformationPass.options;
+}
```

Comment:
Can you remove this one and pass an option to DisentangleCNOTPass to test the states?

Reply:
Good call! https://github.com/PennyLaneAI/catalyst/pull/1154/commits/7330335ee8fe6afc0777a983fcc19210cb0e6e0b